<launch>
  <node name="uvc_camera" pkg="uvc_camera" type="uvc_camera_node" output="screen" >
    <param name="device" value="/dev/video0" />
    <param name="width" value="640" />
    <param name="height" value="480" />
    <param name="fps" value="30"/>
    <param name="frame_id" value="camera" />
    <!-- <param name="camera_info_url" type="string" value="file://$(find PACKAGE_NAME)/cal.yml" />-->
    <param name="camera_info_url" type="string" value="file:///home/{user_name}/.../camera.yaml"/>
  </node>
  <!--
  <node name="camera360" pkg="rpl" type="camera360.py" output="screen" >
    <param name="calibration_path" value="/home/rb/camera_ws/src/rpl/scripts/camera.xml" />
  </node>
-->
  <node name="ar" pkg="rpl" type="ar_detector.py" output="screen"/>
  <node name="get_tmatrix" pkg="rpl" type="get_estimate_coordinates.py" output="screen" >
    <param name="input_files" value="/home/rb/ARenv/Pose_2020-08-07_141152" />
  </node>
<!--
  <node name="record_result" pkg="robot_camera" type="record_result.py" output="screen"/>
    <param name="file_path" value="/home/rb/ARenv"/>
-->
</launch>
